docs/catalog.md

```diff
@@ -5,950 +5,27 @@ In OpenWhisk, a catalog of packages gives you an easy way to enhance your app wi
-- `apnsIosActionKey`: The title for the Action key .
-- `apnsPayload`: Custom JSON payload that will be sent as part of the notification message.
-- `apnsType`: ['DEFAULT', 'MIXED', 'SILENT'].
-- `apnsSound`: The name of the sound file in the application bundle. The sound of this file is played as an alert.
-
-Here is an example of sending push notification from the pushnotification package.
-
-1. Send push notification by using the `sendMessage` action in the package binding that you created previously. Be sure to replace `/myNamespace/myPush` with your package name.
-
-  ```
-  $ wsk action invoke /myNamespace/myPush/sendMessage --blocking --result  -p url https://example.com -p text "this is my message"  -p sound soundFileName -p deviceIds "[\"T1\",\"T2\"]"
-  ```
-
-  ```
-  {
-  "result": {
-  "pushResponse": "{"messageId":"11111H","message":{"message":{"alert":"this is my message","url":"http.google.com"},"settings":{"apns":{"sound":"default"},"gcm":{"sound":"default"},"target":{"deviceIds":["T1","T2"]}}}"
-  },
-  "status": "success",
-  "success": true
-  }
-  ```
-
-### Firing a trigger event on Push activity
-
-The `/whisk.system/pushnotifications/webhook` configures the Push service to fire a trigger when there is a device activity such as device registration / unregistration or subscription / unsubscription in a specified application
-
-The parameters are as follows:
-
-- `appId:` The Bluemix app GUID.
-- `appSecret:` The Bluemix push notification service appSecret.
-- `events:` Supported events are `onDeviceRegister`, `onDeviceUnregister`, `onDeviceUpdate`, `onSubscribe`, `onUnsubscribe`.To get notified for all events use the wildcard character `*`.
-
-The following is an example of creating a trigger that will be fired each time there is a new device registered with the Push Notifications service application.
-
-1. Create a package binding configured for your Push Notifications service with your appId and appSecret.
-
-  ```
-  $ wsk package bind /whisk.system/pushnotifications myNewDeviceFeed --param appID myapp --param appSecret myAppSecret --param events onDeviceRegister
-  ```
-
-2. Create a trigger for the Push Notifications service `onDeviceRegister` event type using your `myPush/webhook` feed.
-
-  ```
-  $ wsk trigger create myPushTrigger --feed myPush/webhook --param events onDeviceRegister
-  ```
+| [/whisk.system/alarms](https://github.com/openwhisk/openwhisk-package-alarms/blob/master/README.md) | Package to create periodic triggers |
+| [/whisk.system/cloudant](https://github.com/openwhisk/openwhisk-package-cloudant/blob/master/README.md) | Package to work with [Cloudant noSQL DB](https://console.ng.bluemix.net/docs/services/Cloudant/index.html) service |
+| [/whisk.system/github](https://github.com/openwhisk/openwhisk-package-catalog/blob/master/packages/github/README.md) | Package to create webhook triggers for [GitHub](https://developer.github.com/). |
```

Comment:
Good catch, it should be
https://github.com/openwhisk/openwhisk-catalog/blob/master/packages/github/README.md